### PR TITLE
Url.php: remove unwanted ?PHPSESSID= URL parameters

### PR DESCRIPTION
**Pull request by nodiscc**
As seen on https://lafibre.info/online/datacenter-dc3-diliad/ and many other websites.

### CODE REVIEW

File: application/Url.php [hunk 1]
```diff
@@ -99,6 +99,7 @@ class Url
         'action_type_map=',
         'fb_',
         'fb=',
+        '?PHPSESSID=',
```
Comment (ArthurHoaro) [inline application/Url.php]: I don't think `?` should be here.

Comment (virtualtam) [inline application/Url.php]: Confirmed, the aim is to get rid of "annoying params" wherever they appear within the URL query ;-)

@nodiscc could you also update the test cases?

**Comment by nodiscc** [comment]
Done + rebased.

**Comment by virtualtam** [comment]
Thanks :)

**Pull request merged**